ExceptionCar.cpp

```diff
@@ -0,0 +1,6 @@
+#include "ExceptionCar.hpp"
+
+const char * MyException::what() const noexcept
+{
+    return "Invalid Value!";
```

Comment:
Dobrze zaimplementowany własny wyjątek :)
Inna opcja to napisanie konstruktora, który jako string przyjmuje wartość. Wtedy funkcja what() która jest zaimplementowana w std::exception zwraca właśnie ten komunikat.